sympy/integrals/tests/test_integrals.py

```diff
@@ -1388,3 +1388,8 @@ def test_issue_14782():
 def test_issue_12081():
     f = x**(-S(3)/2)*exp(-x)
     assert integrate(f, [x, 0, oo]) == oo
+
+def test_issue_15285():
+    y = 1/x - 1
+    f = (4 * y * exp(-2 * y)) / (x**2)
+    assert(integrate(f,[x,0,1]) == 1)
```

Comment:
A few formatting issues to take care of:
- Methods are separated using 2 newline characters
- See usage for spaces around operators, `f = (4*y*exp(-2*y))/(x**2)`
- Spaces after commas, `assert integrate(f, [x, 0, 1]) == 1`